src/http/ratelimiting.rs

```diff
@@ -95,11 +96,11 @@ impl Ratelimiter {
     ///
     /// The bot token must be prefixed with `"Bot "`. The ratelimiter does not
     /// prefix it.
-    pub fn new(client: Arc<Client>, token: impl Into<String>) -> Self {
-        Self::_new(client, token.into())
+    pub fn new(client: Arc<Client>, token: SecretString) -> Self {
+        Self::_new(client, token)
```

Comment:
Now that the `impl Into<String>` is dropped, it is just a proxy to `_new`. I think that `_new` can be dropped, since it's a private function anyway.